src/test/resources/cucumber/pagination/CounterTest.feature

```diff
@@ -1,11 +1,15 @@
 Feature: Counter Deep nested values
 
-  Scenario: Deep Nested value like counter is 10
+  Background:
     Given Counter Data
-    When Key value "counters.[].yearMonth.[].dateCounter.[].likes" is "10"
-    Then Result size should be 2
+    And Default Criteria
 
-  Scenario: Deep Nested value like counter is 20 with non-existant search term
-    Given Counter Data
-    When Complex filtering criteria is "counters.[].yearMonth.[].dateCounter.[].likes=10&*=Vega|vegas"
-    Then Result size should be 0
\ No newline at end of file
+  Scenario Outline: Deep Nested filtering test in counter data
+    When Complex filtering criteria is <criteria>
+    Then Result size should be <expectedSize>
+    And <objId> present in <expectedObjs>
+
+    Examples:
+      | criteria | expectedSize | objId | expectedObjs |
```

Comment:
is this objId or key to be inspected?

Reply:
yes if possible

Reply:
Changed this step to `And Object identifier <objID> is present in <expectedObjsList>`